app/src/main/res/values-de/strings.xml

```diff
@@ -86,9 +86,9 @@
     <string name="setup_fingerprint">Fingerabdruck einrichten</string>
     <string name="fingerprint_confirmed">Fingerabdruck bestätigt!</string>
     <string name="finish">Fertig</string>
-    <string name="password_validation_identical_characters">Keine 3-fachen Zeichen\n</string>
```

Comment:
Can you check if this is on purpose, as far as I can remember this is required

Reply:
yes: https://github.com/gnosis/safe/blob/master/specs/common/password_requirements.rst

Reply:
I mean the newline at the end, I think currently we just concat the error messages as strings and the newline will make sure that they are not all on the same line

Reply:
removed new line at the end of the strings. Strings should not make any assumptions about if and how they will be concatenated